### PR TITLE
release notes for v0.2.0

### PR DESCRIPTION
**Pull request by jklynch**

### CODE REVIEW

File: docs/source/release-history.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Release History
 ***************
 
+v0.1.4 (2021-08-24)
+===================
+* updated documentation for beamline runengine Kafka topic names
```
Comment (mrakitin) [inline docs/source/release-history.rst:7]:
```suggestion
* updated documentation for beamline RunEngine Kafka topic names
```

File: docs/source/release-history.rst [hunk 2]
```diff
@@ -2,6 +2,15 @@
 Release History
 ***************
 
+v0.1.4 (2021-08-24)
```
Comment (mrakitin) [inline docs/source/release-history.rst:5]: Should we release it as v0.2.0? The addition of the new xspress3 support may deserve a minor version bump.

**Review by mrakitin — COMMENTED** [review]
Looks good overall, but have a question about the version.

File: docs/source/release-history.rst (hunk 2 above)
Comment (jklynch) [reply docs/source/release-history.rst:5]: I almost asked before just going with 0.1.4. I'm happy to release as 0.2.0.

Comment (mrakitin) [reply docs/source/release-history.rst:5]: Let's do that.

File: docs/source/release-history.rst [hunk 3]
```diff
@@ -4,7 +4,7 @@ Release History
 
 v0.1.4 (2021-08-24)
```
Comment (mrakitin) [inline docs/source/release-history.rst:5]:
```suggestion
v0.2.0 (2021-08-24)
```

**Review by mrakitin — APPROVED** [review]

**Pull request merged**